modules/acquia_cms_common/src/EventSubscriber/ConfigEventsSubscriber.php

```diff
@@ -197,6 +197,13 @@ public function configSave(ConfigCrudEvent $event) {
         }
       }
     }
+    // Clear cache once acquia purge.
+    else {
+      $config = $event->getConfig();
+      if ($config->getName() == 'purge.plugins') {
```

Comment:
@chandan-singh7929 Add todo here. I don't like the idea of clearing cache in event subscriber. That's not good approach.
cc: @rajeshreeputra

Reply:
Done